File: PB - More Exercises JavaScript/1.0 Simple Operations and Calculations - More Exercises/08. Circle Area and Perimeter.js
function circleAreaAndPerimeter(n) {
    let face = Math.PI * n * n
    let prm =  Math.PI * 2 * n
    return `${face.toFixed(2)}\n${prm.toFixed(2)}`  
}

// console.log(circleAreaAndPerimeter(3)) //28.27   / 18.85

File: PB - More Exercises GO/1.0 Simple Operations and Calculations - More Exercises/08. Circle Area and Perimeter.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n float64
	fmt.Scan(&n)
	face := math.Pi * n * n
	prm :=  math.Pi * 2 * n
	fmt.Printf("%.2f\n%.2f", face, prm)
}

/*
name   :08. Circle Area and Perimeter
input  :3
output :28.27 / 18.85
*/
